rasa/core/policies/form_policy.py

```diff
@@ -96,11 +97,10 @@ def _create_lookup_from_states(
                 lookup[feature_key] = active_form
         return lookup
 
+    # pytype: enable=bad-return-type
+
```

Comment:
```suggestion
```

Reply:
I'd keep the line, since enable closes disable for previous method